source/crud/tests/db/db-aggregate.yml

```diff
@@ -1,11 +1,13 @@
+database_name: &database_name "admin"
```

Comment:
I assume this is only used to denote the database-under-test? I just realized that we don't use `collection_name` in any other CRUD tests -- perhaps because they never cared about what collection they used.

Reply:
Yes, these tests need to use the `admin` database so I suppose the other CRUD tests don't care about which collection they're using.